mono/mini/mini-runtime.c

```diff
@@ -104,6 +104,7 @@
 
 static guint32 default_opt = 0;
 static gboolean default_opt_set = FALSE;
+static char *stats_method_name = NULL;
```

Comment:
I think you'd be better off not making this static, and just using an extern in the header. You also won't need the helper methods in that case. See `sdb_options` in mini-runtime for an example.